cmd/ako-main/main.go

```diff
@@ -140,6 +142,12 @@ func InitializeAKC() {
 		}
 	}
 
+	aviObjCache := avicache.SharedAviObjCache()
+	aviRestClientPool := avicache.SharedAVIClients()
+	if !aviObjCache.IsAviClusterActive(aviRestClientPool.AviClient[0]) {
+		utils.AviLog.Fatalf("Avi Controller Cluster state is not Active, shutting down AKO")
```

Comment:
Can we shutdown using the following  mechanism.
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/99405920a9e3fb701a59206be5ee0f22758902ee/internal/k8s/ako_init.go#L116

Reply:
Is there a benefit in doing that .. we are checking this during boot up, so I thought we can just exit